src/styles/blocks/catalog.scss

```
@@ -17,6 +17,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform 300ms, color 300ms;
```

```suggestion
    transition:
      transform 300ms, 
      color 300ms;
```
This suggestion is just to make code look better. If you want to add more properties later on this way it will look less confusing. But for this small project it is fine of course.